man/systemd.exec.xml

```diff
@@ -285,7 +285,10 @@
         Filesystem</ulink>. It is generally recommended to run most system services with this option set to
         <literal>invisible</literal>. This option is implemented via file system namespacing, and thus cannot
         be used with services that shall be able to install mount points in the host file system
-        hierarchy. It also cannot be used for services that need to access metainformation about other users'
+        hierarchy. Note that the root user is unaffected by this option, so to be effective it has to be used
+        together with <varname>User=</varname></term> or <varname>DynamicUser=yes</varname></term>, and also
```

Comment:
Missing `<term>` in both `<varname>` cases?

Reply:
double copy-pasta!